internal/manager/setup.go

```diff
@@ -445,7 +446,18 @@ func setupLicenseGetter(
 	// we probably want to avoid that long term. If we do have separate toggles, we need an AND condition that sets up
 	// the client and makes it available to all Konnect-related subsystems.
 	if c.Konnect.LicenseSynchronizationEnabled {
-		konnectLicenseAPIClient, err := konnectLicense.NewClient(c.Konnect)
+		setupLog.Info("Creating konnect license client")
```

Comment:
REVIEW: should we add a new cli argument to enable storing Konnect licenses or just enable it when konnect license sync enabled?

Reply:
I think we can add a flag for that, so that we people can control it but I'd be happy to hear others' opinions on this.

I think we could set this on by default, leaving an option for users to disable this.

Reply:
Done.